src/app.ts

```diff
@@ -48,32 +48,42 @@ import UserMetadataController from "./controllers/userMetadataController";
 import { log } from "./utils/logger";
 import checkIP from "./utils/middleware/checkIP";
 
-import NotificationService from "./services/notificationService";
-import { User } from "./types/user";
-import { toExpressHandler } from "./utils/express";
-import {
-  getCurrentBackendVersion,
-  getObjectFromPackageJson
-} from "./utils/package";
-
+import { QueueClient } from "@azure/storage-queue";
 import { withSpid } from "@pagopa/io-spid-commons";
 import { getSpidStrategyOption } from "@pagopa/io-spid-commons/dist/utils/middleware";
 import { isEmpty, StrMap } from "fp-ts/lib/StrMap";
 import { Task } from "fp-ts/lib/Task";
 import { VersionPerPlatform } from "../generated/public/VersionPerPlatform";
 import UserDataProcessingController from "./controllers/userDataProcessingController";
 import MessagesService from "./services/messagesService";
+import NotificationService from "./services/notificationService";
 import PagoPAProxyService from "./services/pagoPAProxyService";
 import ProfileService from "./services/profileService";
 import RedisSessionStorage from "./services/redisSessionStorage";
 import RedisUserMetadataStorage from "./services/redisUserMetadataStorage";
 import TokenService from "./services/tokenService";
 import UserDataProcessingService from "./services/userDataProcessingService";
+import { User } from "./types/user";
+import { getRequiredENVVar } from "./utils/container";
+import { toExpressHandler } from "./utils/express";
+import {
+  getCurrentBackendVersion,
+  getObjectFromPackageJson
+} from "./utils/package";
+import { makeSpidLogCallback } from "./utils/spid";
 
 const defaultModule = {
   newApp
 };
 
+const queueConnectionString = getRequiredENVVar(
+  "SPID_LOG_STORAGE_CONNECTION_STRING"
+);
+const spidQueueName = getRequiredENVVar("SPID_LOG_QUEUE_NAME");
```

Comment:
Move this configuration inside `config.ts` file.
All env vars are handled there.

Reply:
fixed in https://github.com/pagopa/io-backend/pull/611/commits/d1f1f6cc945bdba3deacad2c3b4b2f83a8f71e44